tests/client/test_async_client.py

```diff
@@ -106,7 +106,7 @@ def hello_world() -> typing.Iterator[bytes]:  # pragma: no cover
 
     async with httpx.AsyncClient() as client:
         with pytest.raises(RuntimeError):
-            await client.post(server.url, content=hello_world())
+            await client.post(server.url, content=hello_world())  # type: ignore[arg-type]
```

Comment:
```suggestion
            # We have a type ignore here, because we'd expect this case
            # to *both* be a typing error *and* also raise a RuntimeError
            # if it is violated.
            await client.post(server.url, content=hello_world())  # type: ignore[arg-type]
```